lib/engine/game/g_18_india/step/certificate_selection.rb

```diff
@@ -35,10 +40,8 @@ def may_purchase?(_company)
             false
           end
 
-          def may_choose?(company)
-            return false if selections_completed? && company.owner.nil?
-
-            true
+          def may_choose?(_company)
+            false
           end
```

Comment:
Is this method still needed?

Reply:
This was true before to create a "Choose" button.  I set it to false to prevent the button.  But with the other changes to company view, this works without the method.  I have removed it.